SuperCalculator/src/supercalculator/Tan.java

```diff
@@ -0,0 +1,136 @@
+
+package supercalculator;
+
+import java.util.Scanner;
+/*
+ * Tangent class
+ *
+ *
+ *
+ * 2021-08-07
+ *
+ * Tangent function for super calculator SOEN 6011
+ */
+
+public class Tan {
+
+    /**
+     * Default constructor
+     */
+    public Tan() { }
+
+
+    /**
+     * helper class that calculates cos. Wasn't added to the Math class since it's exclusively used in the tangent
+     * @param  value value in question get cosine
+     * @param isRad Check if value is Radian or Degree
+     */
+    private double cos(final double value, final boolean isRad) {
+        var calculatedValue = (isRad) ? value : Math.convertDegToRad(value);
+        return calculateTaylorsSeries(calculatedValue, 1, 2);
+    }
+
+    /**
+     * helper class that calculates cos. Wasn't added to the Math class since it's exclusively used in the tangent
+     * @param  value value in question get Sine
+     * @param isRad Check if value is Radian or Degree
+     */
+    private double sin(final double value, final boolean isRad) {
+        var sinStartingValue = 3;
```

Comment:
In order to have backward compatibility with previous version of Java which other team mates are using, let's try not to use inference type. Can you please change the var for a static type?